WordPress/gutenberg
# FontSizePicker: Hard deprecate bottom margin

Part of #39358

## What?

Promotes `__nextHasNoMarginBottom` on FontSizePicker to the default behavior.

## Why?

The deprecation grace period has elapsed.

## Testing Instructions

In the Storybook for FontSizePicker, turn on the [margin checker tool](https://github.com/WordPress/gutenberg/assets/555336/f4fcf334-7c27-47ad-acba-e9b3a5f6f734) in the toolbar. The margin-free styles should now be the default, and the `__nextHasNoMarginBottom` prop should not be listed in the props table.

✅ The wrapped FontSizePicker component exported from `@wordpress/block-editor` followed the same deprecation schedule as the canonical FontSizePicker component.
✅ Other usages in the Gutenberg app already are in margin-free mode.
✅ There are no internal components that use FontSizePicker under the hood.

## ✍️ Dev Note

See Dev Note for #58699.
